raddar/crud/crud.py

```diff
@@ -28,6 +29,7 @@ def create_analyze(
     project: schemas.ProjectCreate,
     analyze: schemas.AnalyzeCreate,
     ref_name: str,
+    origin: str
```

Comment:
Here is the follow-up of the previous comment ^^
You can change this `origin` parameter to either be an Enum (I don't know if there is something specific in SQLAlchemy if this parameter is stored in the DB), or you can document the different possibles str values with [`Literal`](https://mypy.readthedocs.io/en/stable/literal_types.html):
```suggestion
    origin: Literal["manual", "github-webhook"]
```

Reply:
(I don't know which one is better, the Enum will probably give you better completion in VSCode/Pycharm and better if your DB supports enums, but could be harder to make it work with SQLAlchemy (again I have no experience so I don't know), whereas the Literal solution is simpler and will definitely work with any DB but you will have less completion (but mypy will still scan your code))

Reply:
I love Literal in this use case